services/EventAPIService.js

```diff
@@ -0,0 +1,71 @@
+const axios = require("axios");
+
+// A service class for all REST API operations
+class EventAPIService {
+  #axiosInstance;
+  #tokenServiceInstance;
+  constructor(tokenService) {
+    // Set Axios baseURL to 100ms API BaseURI
+    this.#axiosInstance = axios.create({
+      baseURL: "https://api.100ms.live/v2",
+      timeout: 3 * 60000,
+    });
+    this.#tokenServiceInstance = tokenService;
+    this.#configureAxios();
+  }
+
+  // Add Axios interceptors to process all requests and responses
+  #configureAxios() {
+    this.#axiosInstance.interceptors.request.use(
+      (config) => {
+        // Add Authorization on every request made using the Management token
+        config.headers = {
+          Authorization: `Bearer ${this.#tokenServiceInstance.getManagementToken()}`,
```

Comment:
Isn't this token supposed to come from the environment?

Reply:
No, a unqiue management token needs to be generated for the API calls bettween RDS backend and 100ms server for authorization and authentication purposes.

Reply:
How will you sign and verify the token?

Reply:
A management token is created by 100ms API itself and verified by themselves.
You can take a look here for how it is signed 
![image](https://github.com/Real-Dev-Squad/website-backend/assets/85296770/a6d6eff2-1c5b-488b-a7d1-0cd7643c370f)

For more you can refer this link to know more about tokens in 100ms : https://www.100ms.live/docs/get-started/v2/get-started/security-and-tokens

Let me know if there is any query regarding the same.

Reply:
Hey @RitikJaiswal75,

Is there anything left with this discussion or we can resolve it?